GitUI/CommandsDialogs/RevisionFileTreeController.cs

```diff
@@ -85,9 +86,9 @@ public TreeNode Find(TreeNodeCollection nodes, string label)
         /// <param name="nodes"></param>
         /// <param name="imageCollection"></param>
         /// <remarks>The method DOES NOT check any input parameters for performance reasons.</remarks>
-        public void LoadChildren(IGitItem item, TreeNodeCollection nodes, ImageList.ImageCollection imageCollection)
+        public async Task LoadChildren(IGitItem item, TreeNodeCollection nodes, ImageList.ImageCollection imageCollection)
         {
-            var childrenItems = _cachedItems.GetOrAdd(item.Guid, _revisionInfoProvider.LoadChildren(item));
+            var childrenItems = _cachedItems.GetOrAdd(item.Guid, await _revisionInfoProvider.LoadChildren(item));
```

Comment:
❗️ Missing `ConfigureAwait` (again I believe the argument here needs to be `true`)